src/main/scala/io/iohk/ethereum/testmode/TestmodeConsensus.scala

```diff
@@ -20,7 +20,7 @@ import io.iohk.ethereum.consensus.pow.validators.ValidatorsExecutor
 class TestmodeConsensus(
     override val vm: VMImpl,
     blockchain: BlockchainImpl,
-    blockchainConfig: BlockchainConfig,
+    var blockchainConfig: BlockchainConfig,
```

Comment:
Is there any other way? I really feel we should avoid adding any more `var`s to the code base.

Reply:
Well, I think we could replace  `TestLedgerWrapper` with some utility class that provide a ledger and also a consensus implementation. We would still have a var somewhere but at least it would be more centralized. I will try something along those lines.

Reply:
Is thread safety a concern btw? This is essentially a shared resource right?

Reply:
Not really. Clearly thread safety is not taken into account in the test service, but retesteth will run the test sequentially anway. There is a way to run tests in parallel but only with several instance of the client. But it wouldn't hurt to put the state in one place and put it behind an AtomicReference.